sources/web/datalab/polymer/components/datalab-sidebar/datalab-sidebar.html

```diff
@@ -68,7 +68,7 @@
       .iron-selected > .sidebar-button > * {
         color: var(--selection-fg-color);
       }
-      @media only screen and (max-width: 970px) {
+      @media only screen and (max-width: 800px) {
```

Comment:
These are actually logically different, and they aren't the same numbers either. These are screen width numbers, the numbers in the javascript are based on the element width, which is smaller.

As far as I know, there isn't an easy way to make this one a constant without using a CSS preprocessor.